modules/nw-openstack-hw-offload-testpmd-pod.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: REFERENCE
+[id="nw-openstack-hw-offload-testpmd-pod_{context}"]
+= A test pod template for clusters that use OVS hardware offloading on OpenStack
+
+The following `testpmd` pod demonstrates Open vSwitch (OVS) hardware offloading on {rh-openstack-first}.
+
+.An example `testpmd` pod
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: testpmd-sriov
+  namespace: mynamespace
+  annotations:
+    k8s.v1.cni.cncf.io/networks: hwoffload1
+spec:
+  runtimeClassName: performance-cnf-performanceprofile <1>
+  containers:
+  - name: testpmd
+    command: ["sleep", "99999"]
+    image: registry.redhat.io/openshift4/dpdk-base-rhel8:v4.9
+    securityContext:
+      capabilities:
+        add: ["IPC_LOCK","SYS_ADMIN"]
+      privileged: true
+      runAsUser: 0
+    resources:
+      requests:
+        memory: 1000Mi
+        hugepages-1Gi: 1Gi
+        cpu: '2'
+      limits:
+        hugepages-1Gi: 1Gi
+        cpu: '2'
+        memory: 1000Mi
+    volumeMounts:
+      - mountPath: /dev/hugepages
+        name: hugepage
+        readOnly: False
+  volumes:
+  - name: hugepage
+    emptyDir:
+      medium: HugePages
+----
+<1> This example assumes that the name of the performance profile is `cnf-performance profile`.
```

Comment:
> Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions.

Perhaps something like:
```suggestion
<1> If your performance profile is not named `cnf-performance profile`, replace that string with correct performance profile name.
```

If you change this line, change it in the next module, too.

Reply:
```suggestion
<1> If your performance profile is not named `cnf-performance profile`, replace that string with the correct performance profile name.
```